adapters/mqtt-vertx-base/src/main/java/org/eclipse/hono/adapter/mqtt/AbstractVertxBasedMqttProtocolAdapter.java

```diff
@@ -325,13 +326,23 @@ final void handleEndpointConnection(final MqttEndpoint endpoint) {
 
     private Future<Device> handleConnectionRequest(final MqttEndpoint endpoint, final Span currentSpan) {
 
+        if (connectionLimitExceeded()) {
+            LOG.info("Connection limit ({}) exceeded, reject connection request", connectionLimit);
```

Comment:
> The log level should be debug FMPOV because this is an expected (and handled) case that will occur thousands of times in production, don't you think?

Probably one wants to see if the protocol adapter regularly hits the limit without constantly logging on debug level. But you are right: logging this for every connection request (above the limit) is probably an overkill.

Reply:
I think that this would be an excellent use case for a Dashboard on top of the metrics back end :-)